src/Library/Plugin/Page.php

```diff
@@ -0,0 +1,240 @@
+<?php
+/**
+ * BoldGrid Library Plugin Page.
+ *
+ * @package Boldgrid\Plugin
+ *
+ * @since SINCEVERSION
+ * 
+ * @author BoldGrid <wpb@boldgrid.com>
+ */
+namespace Boldgrid\Library\Library\Plugin;
+
+/**
+ * Generic page class.
+ *
+ * This class represents a specific page used by the 
+ * Boldgrid\Library\Library\Plugin\Plugin class.
+ *
+ * @since SINCEVERSION
+ */
+class Page {
+
+    /**
+     * Plugin 
+     * 
+     * The Boldgrid\Library\Library\Plugin\Plugin Object
+     * that this page belongs to.
+     *
+     * @since SINCEVERSION
+     * 
```

Comment:
For class properties, the space after `@since` is not needed. For methods, it is.